vendor/engines/bulk_email/app/validators/bulk_email/job_json_field_validator.rb

```diff
@@ -0,0 +1,30 @@
+module BulkEmail
+
+  class JobJsonFieldValidator < ActiveModel::Validator
```

Comment:
Since there's no real user input here, are these validations really necessary? What's the value? They feel like extra overhead without a lot of upside.